docs/v1/P4Runtime-Spec.mdk

```diff
@@ -782,67 +784,146 @@ scheme in an out-of-band fashion.
        (`device_id`, `role_id`), the P4Runtime server shall terminate the stream
        by returning an `INVALID_ARGUMENT` error.
 
-    3. If the max number of clients for the given (`device_id`, `role_id`)
+    3. If `role.config` does not match the "out-of-band" scheme previously
+       agreed upon, the server must return an `INVALID_ARGUMENT` error.
+
+    4. If the max number of clients for the given (`device_id`, `role_id`)
```

Comment:
I think it's more correct to talk about streams here, as there can be clients without an open stream, for example when performing a Read RPC.
```suggestion
    4. If the number of open streams for the given (`device_id`, `role_id`) 
```